examples/imageviewer/mapview-desktop/src/desktopMain/kotlin/example/map/CalcTiles.kt

```diff
@@ -0,0 +1,51 @@
+package example.map
+
+import kotlin.math.ceil
+import kotlin.math.log2
+import kotlin.math.roundToInt
+
+fun example.map.InternalMapState.calcTiles(): List<example.map.DisplayTileAndTile> {
```

Comment:
`example.map.` prefix is not required here (and in other places in that file)

Reply:
Ok, fixed